src/mudlet.cpp

```diff
@@ -5114,9 +5114,14 @@ bool mudlet::desktopInDarkMode()
     return false;
 }
 
-void mudlet::announce(const QString& text, const QString& processing)
+void mudlet::announce(const QString& text, const QString& processing, bool isPlain)
 {
-    mpAnnouncer->announce(text, processing);
+    if (isPlain){
+        mpAnnouncer->announce(text, processing);
+    } else {
+        QTextEdit convertor(text);
```

Comment:
This seems expensive to create a visual widget that can display rich text every time you need to read some text... how about using a static `QTextDocument` instead that is created once?